libraries/ESP8266WiFiMesh/src/ESP8266WiFiMesh.h

```diff
@@ -24,52 +24,278 @@
+   *
+   * @param newNodeID The node id to change to.                        
+   */
+  void setNodeID(const String &newNodeID);
+  
+  /**
+   * Change the SSID (mesh name + node id) used by this ESP8266WiFiMesh instance. 
+   * Will also change the AP SSID if this ESP8266WiFiMesh instance is the current AP controller.
+   *
+   * @param newMeshName The mesh name to change to. Will be the SSID prefix.                    
+   * @param newNodeID The node id to change to. Will be the SSID suffix.
+   */  
+  void setSSID(const String &newMeshName, const String &newNodeID);
+
+  String getMessage();
+
+  /**
+   * Set the message that will be sent to other nodes when calling attemptTransmission.
+   * 
+   * @param newMessage The message to send.
+   */
+  void setMessage(const String &newMessage);
+
+  /**
+   * If AP connection already exists, and the initialDisconnect argument is set to false, send message only to the already connected AP.
+   * Otherwise, scan for other networks, send the scan result to networkFilter and then transmit the message to the networks found in connectionQueue.
+   *
+   * @param message The message to send to other nodes. It will be stored in the class instance until replaced via attemptTransmission or setMessage.
+   * @param concludingDisconnect Disconnect from AP once transmission is complete.
+   * @param initialDisconnect Disconnect from any currently connected AP before attempting transmission.
+   * @param noScan Do not scan for new networks and do not call networkFilter function. Will only use the data already in connectionQueue for the transmission.
+   * @param scanAllWiFiChannels Scan all WiFi channels during a WiFi scan, instead of just the channel the ESP8266WiFiMesh instance is using.
+   *                               Scanning all WiFi channels takes about 2100 ms, compared to just 60 ms if only channel 1 (standard) is scanned.
+   *                               Note that if the ESP8266 has an active AP, that AP will switch WiFi channel to match that of any other AP the ESP8266 connects to.
+   *                               This can make it impossible for other nodes to detect the AP if they are scanning the wrong WiFi channel.
+   */
+  void attemptTransmission(const String &message, bool concludingDisconnect = true, bool initialDisconnect = false, bool noScan = false, bool scanAllWiFiChannels = false);
+
+  /**
+   * If any clients are connected, accept their requests and call the requestHandler function for each one.
+   */
+  void acceptRequest();
+
+  /**
+   * Set a static IP address for the ESP8266 and activate use of static IP.
+   * The static IP needs to be at the same subnet as the server's gateway.
+   */
+  void setStaticIP(const IPAddress &newIP);
+  
+  IPAddress getStaticIP();
+  void disableStaticIP();
+
+  /**
+   * An empty IPAddress. Used as default when no IP is set.
+   */
+  static const IPAddress emptyIP;
+
+  static String uint64ToString(uint64_t number, byte base = 16);
```

Comment:
It looks like I wasn't clear about this. I meant move this completely out of the class. These two functions have nothing to do with the mesh nodes, or wifi, or even with the ESP really. They should be free standing functions. 
But let's leave that for a subsequent cleanup.